src/lib/response-triggers/removeuser.ts

```diff
@@ -0,0 +1,101 @@
+import Trigger from '../../models/trigger';
+import Config from '../config-interface';
+import { DbUsers, DbLocation, PockyDB } from '../database/db-interfaces';
+import { MessageObject } from 'webex/env';
+import { Role, UserRow } from '../../models/database';
+import { Command } from '../../models/command';
+import constants from '../../constants';
+import xmlMessageParser from '../parsers/xmlMessageParser';
+import { Logger } from '../logger';
+
+export default class RemoveUser extends Trigger {
+	private config : Config;
+	private dbUsers : DbUsers;
+	private dbLocation : DbLocation;
+	private pockyDb : PockyDB;
+
+	constructor(config : Config, dbUsers : DbUsers, dbLocation : DbLocation, pockyDb: PockyDB) {
+		super();
+
+		this.config = config;
+		this.dbUsers = dbUsers;
+		this.dbLocation = dbLocation;
+		this.pockyDb = pockyDb;
+	}
+
+	isToTriggerOn(message : MessageObject) : boolean {
+		if (!(this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.RemoveUser))) {
+			return false;
+		}
+
+		let parsedMessage = xmlMessageParser.parseNonPegMessage(message);
+		return parsedMessage.botId === constants.botId && parsedMessage.command.toLowerCase().startsWith(Command.RemoveUser);
+	}
+
+	async createMessage(message : MessageObject) : Promise<MessageObject> {
+		let parsedMessage = xmlMessageParser.parseXmlMessage(message);
+		if (parsedMessage.length === 3) {
+			if (parsedMessage[1].text().trim().toLowerCase() !== Command.RemoveUser
+				|| parsedMessage[2].name() !== 'spark-mention' || parsedMessage[2].attr('data-object-type').value() !== 'person') {
```

Comment:
Why can't you remove bots?

Reply:
`data-object-type="person"` doesn't actually mean it's a real person, it just means that it is a mention of an individual bot or person